pkg/cloud/scope/machine.go

```diff
@@ -180,29 +180,66 @@ func (m *MachineScope) SetAnnotation(key, value string) {
 	m.AWSMachine.Annotations[key] = value
 }
 
+// IsCloudInitEncrypted returns the computed value of whether or not
+// userdata should be stored using AWS Secrets Manager. Defaults to true
+// for control plane machines.
+func (m *MachineScope) IsCloudInitEncrypted() bool {
+	if m.AWSMachine.Spec.CloudInit != nil {
+		return m.AWSMachine.Spec.CloudInit.Encrypted
+	}
+	return false
+}
+
+// GetSecretARN returns the Amazon Resource Name for the secret belonging
+// to the AWSMachine in AWS Secrets Manager
+func (m *MachineScope) GetSecretARN() *string {
```

Comment:
Should this return a non-pointer so callers can avoid the nil check and instead just compare against ""?